third-party/snyk/snapshot.json

```diff
@@ -47,10 +47,10 @@
       {"id":"SNYK-JS-DOJO-174933","severity":"medium","semver":{"vulnerable":["<1.2.0"]}},
       {"id":"SNYK-JS-DOJO-72305","severity":"medium","semver":{"vulnerable":["<1.14"]}},
       {"id":"npm:dojo:20180818","severity":"medium","semver":{"vulnerable":["<1.10.10",">=1.11.0 <1.11.6",">=1.12.0 <1.12.4",">=1.13.0 <1.13.1"]}},
-      {"id":"npm:dojo:20160523","severity":"medium","semver":{"vulnerable":["<1.1.0"]}},
+      {"id":"npm:dojo:20160523","severity":"medium","semver":{"vulnerable":["<0.0.0"]}},
```

Comment:
Now a <0.0.0 along with the noop loadsh change? what is up with you snyk-bot

Reply:
ya shouldnt this be deleted?

Reply:
hey @aviadatsnyk, these seem like potential mistakes?

Reply:
bump, @aviadatsnyk we got another PR of these noop updates today #11179

Reply:
Sorry for missing this, @patrickhulce . Usually @lirantal is better than me for these things :)
This is our way of saying it was deemed not a vulnerability, while still letting you know it existed once, instead of just pulling the carpet from under your feet.. This is the equivalent of the empty range, and so no user will ever have this vulnerability "found" in their scan. I'm not sure it makes sense for your usecase, but this is how we do it to support other ones.
Is this a problem?

Reply:
ah gotcha, thanks for the explanations @aviadatsnyk! 

These changes all make sense from the perspective of a shared database 👍  

`<0.0.0` we can filter out on our end in https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/scripts/cleanup-vuln-snapshot.js so that one's not a problem at all. 

From the Lighthouse perspective it's unfortunate to invest review attention on PRs with no user-impact, so the `<4.17.16` type of change ideally wouldn't trigger the bot and just be lumped in with the next real change. Not sure how we could enforce that though, so I'm fine with this.